pgrx/src/pg_sys.rs

```diff
@@ -39,7 +39,7 @@ pub unsafe fn rt_fetch(index: Index, range_table: *mut List) -> *mut RangeTblEnt
 #[inline]
 pub unsafe fn planner_rt_fetch(index: Index, root: *mut PlannerInfo) -> *mut RangeTblEntry {
     unsafe {
-        if (*root).simple_rte_array != core::ptr::null_mut() {
+        if !(*root).simple_rte_array.is_null() {
```

Comment:
I tend to dislike the `!value.access.access.is_null()` style because I lose track of the `!` by the time I get to the test method call.